src/Fixer/LanguageConstruct/NullableTypeDeclarationFixer.php

```diff
@@ -251,6 +251,10 @@ private function normalizeNullableType(Tokens $tokens, TypeAnalysis $typeAnalysi
         $type = $typeAnalysis->getName();
 
         if (!str_contains($type, '|') && !str_contains($type, '&')) {
+            if ('null' === $tokens[$typeAnalysis->getStartIndex()]->getContent()) {
```

Comment:
I am not sure about this:

- you check for `null|*` but not for `*|null`
- this condition is added within the condition that explicitly checks whether `|` is _not_ present

Reply:
In 254 line, can certain the `$type` is a standalone type, and then I made sure that if it was `null`, I wouldn't handle it.

Reply:
I think the fix for this should be in the `isTypeNormalizable()` method, checking if `$name === 'null'`.